examples/agentOnly/helmValues-kubecost-aws.yaml

```
@@ -0,0 +1,28 @@
+global:
+  ## clusterId is used to identify the cluster in the FinOps Agent
+  ## when using the same cluster names in different regions, we suggest appending the region to the clusterId
+  clusterId: globally-unique-cluster-id
+  federatedStorage:
+    ## Use the existingSecret when a access key and secret is used to access the federated storage.
+    ## the 'config' can be used when using assumed roles.
```

I think the comment about assumed roles can lead to user confusion. It may be easier just to keep it simple like this.

```suggestion
    # A federated storage config can be provided via an existingSecret or via a string in helm values.
    # If both are provided, the string in helm values take precedence.
```